lib/rich_text.rb

```diff
@@ -76,11 +76,13 @@ def sanitize(text)
     end
 
     def linkify(text, mode = :urls)
-      if text.html_safe?
-        Rinku.auto_link(text, mode, tag_builder.tag_options(:rel => "nofollow noopener noreferrer")).html_safe
-      else
-        Rinku.auto_link(text, mode, tag_builder.tag_options(:rel => "nofollow noopener noreferrer"))
-      end
+      link_attr = tag_builder.tag_options(:rel => "nofollow noopener noreferrer")
+      Rinku.auto_link(ERB::Util.html_escape(text), mode, link_attr) do |url|
+        %r{^https?://([^/]*)(.*)$}.match(url) do |m|
```

Comment:
Maybe put the www subdomain check into the regex:
```suggestion
        %r{^https?://(?:www\.)?([^/]*)(.*)$}.match(url) do |m|
```
Otherwise linkify_hosts / linkify_domains could get unwieldy long when including all the domains listed in [osm.wiki/wiki/Domain names](//osm.wiki/wiki/Domain_names):
```yaml
["openstreetmap.com", "openstreetmap.net", "openstreetmaps.org", "openmaps.org", "osm.org"]
```

Reply:
It can only get 2x as long. And you copy the domain names from the chef repo. If the names can be managed there, they can be managed here.

Reply:
The settings file can always use the multiline array syntax if the list is too unwieldy.